extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/common/runtime/BeanUtils.java

```diff
@@ -82,4 +92,9 @@ public static String getDatabaseName(MongoEntity mongoEntity, String clientBeanN
                 "The database attribute was not set for the @MongoEntity annotation neither was the database property configured for the named Mongo Client (via 'quarkus.mongodb.%s.database')",
                 mongoEntity.clientName()));
     }
+
+    public static Optional<String> getDatabaseNameFromResolver() {
```

Comment:
This can probably just be one method combined with `getValueFromArcInstance` since no one else is calling that.

Reply:
The reason is the same of previous commen. The initial idea was submit more than one resolver and I removed the other ones that was using this common method. Currently, really doesn't make senses and I'll fix it